openlayers-amplify/vite.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from 'vite'
+
+export default defineConfig({
+  resolve: {
+    alias: {
+      './runtimeConfig': './runtimeConfig.browser',
+    },
+  },
+  define: {
+    'window.global': {}
+  },
+  build: {
+    target: 'esnext'
+  },
+})
```

Comment:
Please include the `server` configs from [this other sample](https://github.com/aws-samples/amazon-location-samples/blob/main/maplibre-gl-js-vue-amplify/vite.config.js#L14-L17), it'll help with customers who are running this on browser-based IDEs like Cloud9.

Reply:
Fixed.